src/psyclone/tests/domain/gocean/transformations/gocean_opencl_trans_test.py

```diff
@@ -51,17 +51,40 @@
     ScalarType, ArrayType, INTEGER_TYPE, REAL_TYPE
 from psyclone.tests.gocean1p0_build import GOcean1p0OpenCLBuild
 from psyclone.tests.utilities import Compile, get_invoke
-from psyclone.transformations import TransformationError
-
-# PSyclone API under test
-API = "gocean1.0"
+from psyclone.transformations import TransformationError, \
+    KernelImportsToArguments
 
 
 @pytest.fixture(scope="function", autouse=True)
 def setup():
-    '''Make sure that all tests here use gocean1.0 as API.'''
+    '''Make sure that all tests here use the GOcean API and include the
+    gocean test_files directory (as some modules are imported from there
+    in the examples) and that we clean up the config file at the end of
+    the tests.'''
+
     Config._instance = None
+    # Each os.path.dirname() move up in the folder hierarchy
+    filepath = os.path.join(
+                   os.path.join(
+                       os.path.dirname(
+                           os.path.dirname(
+                               os.path.dirname(
+                                   os.path.dirname(
+                                       os.path.abspath(__file__))))),
+                       "test_files"),
+                   "gocean1p0")
+
     Config.get().api = "gocean1.0"
+    Config.get()._include_paths = [filepath]
+    yield()
+    # At the end of all tests make sure that we wipe the Config object
```

Comment:
Should that be "every test" instead of "all tests" since the scope of this fixture is 'function'?